src/pages/currency-trading/forms/TransactionForm.js

```diff
@@ -416,28 +462,27 @@ export default function TransactionForm({ recordId, labels, maxAccess, plantId,
         status: record.headerView.status
       })
 
-      CashFormik.setValues({ rows: record.cash })
     }
   }
 
-  // const [plantId, setPlantId] = useState(null)
 
   const { userId } = JSON.parse(window.sessionStorage.getItem('userData'))
```

Comment:
Sometimes sessionstorage isn't populated. Make sure you test wigh when remember me is used when signing in.